scripts/addVersion.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Yeah, probably more `scripts/util` or somewhere that explicitly marks it as tooling, and not a program.
👍